frontend/ui/data/css_tweaks.lua

```diff
@@ -1026,11 +1044,54 @@ This tweak can be duplicated as a user style tweak when books contain footnotes
 {
     -cr-hint: footnote-inpage;
     margin: 0 !important;
-    font-size: 0.8rem !important;
+}
+*, autoBoxing {
+    -cr-hint: late;
+    -cr-only-if: inpage-footnote;
+        font-size: 0.8rem !important;
+}
+*, autoBoxing {
+    -cr-hint: late;
+    -cr-only-if: extended-inpage-footnote;
+        font-size: 0.8rem !important;
 }
             ]],
             separator = true,
         },
+        {
+            title = _("In-page footnote extension"),
+            {
+                id = "following-footnote-inpage_all",
+                conflicts_with = function(id) return util.stringStartsWith(id, "following-footnote-inpage_") end,
+                title = _("Extend footnote content until next entry"),
+                description = _([[
+Extend in-page footnotes shown at the bottom of pages to include text up to the next footnote.
+This is needed when books don't correctly mark all text that belongs to the footnote.]]),
```

Comment:
may be / might be (because who are we to force inpage footnotes to the use ? :)) I think that's the tone we have in other tweak descriptions.

Reply:
done